index.html

```diff
@@ -62,6 +225,27 @@ <h1>I’m Lilian
         </li>
       </ul>
     </div>
+    <div id="iteme-7"><button class="buttonresume">Get my Resume</button></div>
+    <div class="my-accordion">
+      <ul>
+        <li class='menu'>Langguages<span class='arrow rarrow'></span><span class='arrow darrow'>-</span>
+
+
+        <li class='menu'><a href=""><img src="/images/Ellipse1.png" alt="javascript">JavaSript</a></li>
+        <li class='menu'><a href=""><img src="/images/Ellipse2.png" alt="html">HTML</a></li>
+        <li class='menu'><a href=""><img src="/images/Ellipse3.png" alt="css">CSS</a>
+        </li>
+
+
+        <li class='menu'>Frameworks<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
+
+
+        <li class='menu'>Skills<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
+      </ul>
+
+
```

Comment:
Reduce the space between the got resume btn and the languages also add background color to each on of the languages  and align the text to the icon
#### Original design
![right3](https://user-images.githubusercontent.com/58376020/125164391-43914200-e192-11eb-9a8d-64ede0dce631.png)

#### Your project
![wrong3](https://user-images.githubusercontent.com/58376020/125164398-4db34080-e192-11eb-9dbd-eff5601c63a3.png)

Reply:
Resolved